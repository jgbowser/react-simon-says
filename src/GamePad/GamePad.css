.game_pad {
  flex: 1;
  width: 140px;
  height: 140px;
  margin: 10px;
  border-radius: 10px;
}