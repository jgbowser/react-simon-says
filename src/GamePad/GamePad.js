import React from 'react'
import './GamePad.css'

export default class GamePad extends React.Component {
  render() {
    return (
      <div className="game_pad" style={{backgroundColor: this.props.color}}>

      </div>
    )
  }
}