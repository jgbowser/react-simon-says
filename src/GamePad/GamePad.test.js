import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import GamePad from './GamePad'

describe('GamePad Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<GamePad />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
  it('renders a green background when passed green as a prop', () => {
    const tree = renderer
      .create(<GamePad color="green" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})