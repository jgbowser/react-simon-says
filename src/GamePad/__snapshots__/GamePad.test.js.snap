// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GamePad Component renders a green background when passed green as a prop 1`] = `
<div
  className="game_pad"
  style={
    Object {
      "backgroundColor": "green",
    }
  }
/>
`;
